dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/ResourceLink.java

```diff
@@ -43,22 +42,25 @@
     private final String resourceLinkUriAsString;
 
     private final String mimeType;
+    private final String fieldVar;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'fieldVar' could be replaced by a local variable.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)